.github/workflows/snap.yaml

```diff
@@ -0,0 +1,31 @@
+name: snap
+on:
+  push:
+    branches:
+      - developer-guide-part2-answer
+  pull_request:
+    branches:
+      - developer-guide-part2-answer
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  snap:
+    uses: ubuntu-robotics/snap_workflows/.github/workflows/build-install-test-snap.yaml@main
+    with:
+      ubuntu-image: ubuntu-20.04
+      branch-name: developer-guide-part2-answer
+      snap-name: turtlebot3c
+      snap-install-args: --devmode
+      test-script: |
+                    #!/bin/bash
+
+                    sudo snap services turtlebot3c | grep -E "map-cleaner|timer-activated"
+                    if [ $? -eq 0 ]; then
+                        echo "map-cleaner deamon exists"
+                    else
+                      echo "map-cleaner does not exist"
+                      exit 1
+                    fi
+
+                    sudo snap start turtlebot3c.map-cleaner
```

Comment:
Can't we create a dummy yaml and pgm file, call the map cleaner and verify that it was properly deleted (the current version of the code removes even recent files)

Reply:
ah yes, if it deletes them, i'll add that now